zajecia4/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="pl">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Moja pierwsza strona</title>
+    </head>
+
+    <body>
+        <header>
+            <img src="https://images.vexels.com/media/users/3/140692/isolated/lists/72d1f12edf758d24f5b6db73bac4f297-linux-logo.png" alt="obrazek.png"
+                width="100" 
+                height="100"/>
+            <nav>
+                <ul>
+                    <li><a href="./kontakt/index.html">Kontakt</a></li>
+                    <li><a href="./o_mnie/index.html">O mnie</a></li>
+                </ul>
+            </nav>  
+        </header>
+
+        <main>
+            <section>
+                <h1>Strona główna</h1>
+                <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. 
+                Fugiat ratione omnis perferendis doloremque voluptas blanditiis 
+                maxime sunt distinctio minus explicabo? Quia ex pariatur beatae suscipit 
+                obcaecati laudantium dolores est voluptatibus.</p>
+            </section>
+
+            <section>
+                <h3>Pomagamy w rozwoju firm!</h3>
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+                Vel, recusandae enim. Velit aut corporis aliquid possimus rerum, tempore sunt? Consectetur
+                nisi voluptatibus officia debitis, quasi consequatur culpa sed molestiae nulla.</p>
+            </section>
+
+            <section>
+                <h3>Cennik usług</h3>
+                <table border="2" width="30%">
+
+                    <tr>
+                        <th>USŁUGA</th>
+                        <th>CENA</th>
+                    </tr>
+                    <tr>       
+                        <td>SEO</td>
+                        <td>Uśmiech</td>
+                    </tr>
+                    <tr>
+                        <td>Marketing</td>
+                        <td>Wystarczy poprosic :)</td>
+                    </tr>  
+                </table>
+            </section>
+
+            <section>
+                <h4>Dowiedz się o nas więcej!</h4>
```

Comment:
jesli uzywasz naglowkow jako tytulow sekcji, to kazda sekcja powinna miec ten samy typ naglowka, najlepiej h2, z hierarchia schodzisz w dol juz w obrebie samej sekcji